crates/cli-support/src/js/mod.rs

```diff
@@ -865,7 +878,7 @@ impl<'a> Context<'a> {
 
         format!(
             "\
-                function init(module_or_path, maybe_memory) {{
```

Comment:
Although only used in some situations, this is used above if `mem.import.is_some()` holds true. I imagine though you got a warning for an unused argument? Perhaps we could only include the argument if `mem.import.is_some()`?

Reply:
Oooppss :S This was a totally overlooked decision